src/main/java/com/github/tomakehurst/wiremock/client/ResponseDefinitionBuilder.java

```diff
@@ -66,7 +67,13 @@ public static ResponseDefinitionBuilder like(ResponseDefinition responseDefiniti
             ? Parameters.from(responseDefinition.getTransformerParameters())
             : Parameters.empty();
     builder.wasConfigured = responseDefinition.isFromConfiguredStub();
-    return builder;
+
+    ProxyResponseDefinitionBuilder proxyResponseDefinitionBuilder = new ProxyResponseDefinitionBuilder(builder);
```

Comment:
I think we should do this conditionally, in order to maintain the same behaviour when building from scratch, so e.g.

```java
if (builder.proxyBaseUrl != null) {
  // Construct the ProxyResponseDefinitionBuilder
}

// Otherwise keep the builder as-is and return it.
```